test-framework/ui/src/main/java/org/keycloak/testframework/ui/page/AbstractPage.java

```diff
@@ -26,6 +30,10 @@ public String getCurrentPageId() {
         return body.getAttribute("data-page-id");
     }
 
+    public String getPageTitle() {
```

Comment:
Looking at the usage of getPageTitle it's asserting that it's on a specific page, so perhaps just do `loginPage.assertCurrent()` instead in the test of asserting the page title?